src/openrct2/core/Guard.cpp

```diff
@@ -78,15 +78,11 @@ namespace Guard
         Console::Error::WriteLine("Version: %s", gVersionInfoFull);
 
         // This is never freed, but acceptable considering we are about to crash out
-        utf8* formattedMessage = nullptr;
+        std::string formattedMessage;
         if (message != nullptr)
         {
             formattedMessage = String::Format_VA(message, args);
-            Console::Error::WriteLine(formattedMessage);
-        }
-
-        if (formattedMessage != nullptr)
-        {
+            Console::Error::WriteLine(static_cast<std::string_view>(formattedMessage).data());
```

Comment:
Why is this cast necessary? Just passing the string should be enough. It'll implicitly cast to string_view anyway.

Reply:
Addressed by removing static_cast and replacing data with c_str